generate-story.py

```diff
@@ -14,10 +14,10 @@ def get_input(word_type, first_occurrence, tense=None):
 
 noun_1 = get_input("noun", True)
 noun_2 = get_input("noun", False)
-adjective_1 = input("Please enter an adjective: ")
-adjective_2 = input("Please enter another adjective: ")
-adjective_3 = input("Please enter another adjective: ")
-verb = input("Please enter a verb in the past tense: ")
+adjective_1 = get_input("adjective", True)
+adjective_2 = get_input("adjective", False)
+adjective_3 = get_input("adjective", False)
+verb = get_input("verb in the past tense", True)
```

Comment:
Nice try!

We need to pass an argument to the "tense" parameter.

When passing the argument, think about the type of data that the parameter is expecting.

Also remember that unless we put text in quotes, python will think we are referring to a variable name, and look around for where that variable is defined. If it's not defined anywhere, it will throw a NameError.

Reply:
Resolved.